views/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@oat-sa/tao-items",
+  "version": "0.1.0",
+  "license": "GPL-2.0",
+  "description": "taoItems frontend modules",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oat-sa/extension-tao-items.git"
+  },
+  "dependencies": {
+    "@oat-sa/tao-item-runner": "0.3.0",
+    "@babel/polyfill": "^7.4.4"
```

Comment:
Why do you need the `@babel/polyfill here`?

Reply:
no it's not needrf